node/src/node_configurator/configurator.rs

```diff
@@ -817,11 +818,15 @@ impl Configurator {
         string_number: String,
         config: &mut Box<dyn PersistentConfiguration>,
     ) -> Result<(), (u64, String)> {
-        let block_number = match string_number.parse::<u64>() {
-            Ok(num) => num,
-            Err(e) => return Err((NON_PARSABLE_VALUE, format!("start block: {:?}", e))),
+        let block_number_opt = if "none".eq_ignore_ascii_case(&string_number) {
```

Comment:
`eq_ignore_ascii_case()` looks to me like it's part of the Rust standard library, isn't it? Normally I try not to write tests that only test external library code already having its own tests. Do you agree?

Reply:
I agree with your take on this but it isn't really addressing what I was poking into. I can go along with you that sometimes my opinions are extreme, as well as here: I just pointed out in my old comment that your only test doesn't cover the behaviour which was added by incorporating the `eq_ignore_ascii_case`. It really is extreme because my reasoning stem from "if somebody went and rewrote the place with some different code, yet accepting "none", there is no guarantee it would still be code supporting also the other letter-casing. Here it is slightly hilarious, but it raises the question whether it is important that this extra little functionality is in there.

Okay, restating myself: my concern was that the test cannot tell apart, if the production code uses 
```
let block_number_opt = if "none".eq_ignore_ascii_case(&string_number)...
```
or simply
```
let block_number_opt = if  string_number == "none" {...
```

You see what I mean?